app/views/idv/review/new.html.erb

```diff
@@ -38,11 +33,6 @@
   <div class="text-right margin-top-neg-4 margin-bottom-4">
     <%= link_to(t('idv.forgot_password.link_text'), idv_forgot_password_url, class: 'margin-left-1') %>
   </div>
-  <%= render AccordionComponent.new do |c| %>
-    <% c.with_header { t('idv.messages.review.intro') } %>
-    <%= render 'shared/pii_review', pii: @applicant,
-                                    phone: PhoneFormatter.format(@applicant[:phone]) %>
```

Comment:
Do we still need to set `@applicant` in the controller / specs, or is it now unused?

Reply:
Good catch, fixed in https://github.com/18F/identity-idp/pull/8508/commits/d96dd21e95af2077d9254afa49ba75c8075f9e05